singer-io/tap-bing-ads
# Testing/start date

# Description of change
Test start date is respected by incremental streams. Account for all report streams with data including `ad_group_performance_report` and `campaign_performance_report` which require field selection to account for exclusion rules on microsoft's end.

# Manual QA steps
 - Verify the undocumented (by bing) exclusion fields accounted for in tap do result in a failure to return report data.
 
# Risks
 - None, new test added to repo, additions to tests/base.py.
 
# Rollback steps
 - revert this branch
